communities/dijon.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Dijon Software Craftsmanship",
+  "url": "https://softwarecrafters21.slack.com",
```

Comment:
People can not join your Slack team through that URL. Do you have a Meetup page or a slackinviter where people can sign up?